### PR TITLE
Added first Automation test

### PR DESCRIPTION
**Pull request by marikmAS**
Search test as per TP added

### CODE REVIEW

File: test/automation/search_automation_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	cp.ExpectExitCode(1)
+
+}
+
+func (suite *SearchAutomationTestSuite) TestSearch_OutProject() {
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("search", "flask")
+	cp.ExpectLongString("Language must be provided by flag or by running this command within a project.")
```
Comment (Naatan) [inline test/automation/search_automation_test.go:47]: I would recommend shrinking this assertion to something like:

```
cp.ExpectLongString("Language must be provided by flag")
```

Two reasons for this:

1. The longer the assertion the more likely we'll run into text wrapping issues in the assertion
2. The longer the assertion the more likely it is to break if we make copy changes

File: test/automation/search_automation_test.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	cp.ExpectExitCode(1)
+
+}
+
+func (suite *SearchAutomationTestSuite) TestSearch_Flask() {
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	url := "https://platform.activestate.com/qamainorg/public?branch=main&commitID=32e543ee-b6ab-4f59-9b28-ad830ec6980e"
+	suite.Require().NoError(fileutils.WriteFile(filepath.Join(ts.Dirs.Work, "activestate.yaml"), []byte("project: "+url)))
```
Comment (Naatan) [inline test/automation/search_automation_test.go:57]: Was this necessary? IIRC you can use `state search` without a project.

**Review by Naatan — APPROVED** [review]
Looking good! Nothing alarming on my end, just flagged some things that you might want to have a closer look at.

File: test/automation/search_automation_test.go (hunk 2 above)
Comment (marikmAS) [reply test/automation/search_automation_test.go:57]: Using search without `--language` should generate an error in some cases including this one.
Another use is verification which in all situations it provides required info.

**Pull request merged**